partner/WHMCS/README.md

```diff
@@ -0,0 +1,28 @@
+
+### Installation/Setup:
+To install the module upload the ```/jetpack``` folder to the ```/modules/servers``` directory of your WHMCS installation. A server is not required for the module to function so the module will not be listed in the WHMCS Servers under the Setup tab for Products/Servers. Once uploaded to WHMCS the module is ready for use and can be set up when making a new product in WHMCS.
+
+### Usage:
+To use the module create a new product in one of your existing product groups. As part of the product creation process in the Module Settings tab select the “Jetpack by Automattic” module to get started. You will need to have an established Jetpack Hosting Partner account and your client_id and client_secret will be requested in order to create the product and use the module so have these handy when creating the new product. 
+
+When setting up the module 4 custom fields will need to be added in order for the module to work correctly. These fields are verified whenever the module is utilized to create an account. The first 3 fields are required in order to provision a Jetpack plan. They can optionally be shown on the order form to allow your users to include these when checking out and assist with easier provisioning. This is the “Show on Order Form” option when creating custom fields within WHMCS. The last field is used to store details on the response from a provisioning attempt and should not be shown on the order form. When a user is provisioning a new plan this will typically be a URL that they will need to connect the plan you have provisioned for them to their https://wordpress.com account and get started with their Jetpack plan. Please use the field names exactly as shown below as these will be validated when the module is used for provisioning. The fields required are as follows:
+
+
+| Field Name | Field Type | Required | Show on Order Form |
+| ------ | ------ | ------ | ------ |
+| Site URL | Text Box | Yes | Optional |
+| Local User | Text Box | Yes | Optional |
+| Plan | Drop Down | Yes | Optional |
+| jetpack_provisioning_details | Text Area | No | No |
+
+As part of the the product creation for WHMCS you may use any of the 4 options under Module Settings for setting up the product. In the Clients Tab of WHMCS admin under Products/Services the module currently provides functionality for Create and Terminate for Jetpack plans. If you do not setup the product to be automatically provisioned on checkout you will be able perform either of these actions for your users. 
+
```

Comment:
> In the Clients Tab of WHMCS admin under Products/Services the module currently provides functionality for Create and Terminate for Jetpack plans. If you do not setup the product to be automatically provisioned on checkout you will be able perform either of these actions for your users.

I've set my product up so that it doesn't automatically provision.

![screen shot 2018-08-23 at 11 07 26 am](https://user-images.githubusercontent.com/1126811/44537605-da3c8400-a6c4-11e8-93f9-75c87b860289.png)

But, I don't see the ability to create and terminal Jetpack plans in the clients tab.

<img width="238" alt="screen shot 2018-08-23 at 11 06 08 am" src="https://user-images.githubusercontent.com/1126811/44537669-035d1480-a6c5-11e8-926c-e4bf592cf46f.png">

Reply:
We chatted about this earlier today and I've updated the documentation to try to be clearer that the module is intended to manage and provision plans that are setup from a product purchase by the hosting partners client and a user checkout needs to be done first to see the product in the clients product listing. Let me know if the updates to the README would have helped or if you think further tweaking is needed 🙂